index.html

```diff
@@ -55,29 +55,73 @@
-        <p><a class="btn btn-primary btn-lg" href="#" role="button">Learn more &raquo;</a></p>
+        <div class="row">
+          <h1 class="text-center">Popular Categories</h1>
+        </div>
+        <div class="row">
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+        </div>
       </div>
     </div>
 
     <div class="container">
       <!-- Example row of columns -->
       <div class="row">
-        <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+        <h1 class="text-center">New &amp; Popular</h1>
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object" src="images/placeholder-200.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="media-body">
+            <h4 class="media-heading">Happy Valley</h4>
+            <p class="poem-info"><span class="author">Alice Lyons</span><span class="category">Nature</span><span class="rating">&#9734;&#9734;&#9734;&#9734;</span><span class="add-favorite pull-right glyphicon glyphicon-plus"></span></p>
+            <p class="summary">The brook is this mix of roar &amp; hiss as if God has managed to scalpel a section of tempest &amp; clothespin it in the woods over there always draped in the trees...</p>
+          </div>
+        </div>
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object" src="images/placeholder-200.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="media-body">
+            <h4 class="media-heading">And Then It Was Less Bleak Because We Said So</h4>
+            <p class="poem-info"><span class="author">Wendy Xu</span><span class="category">Love</span><span class="rating">&#9734;&#9734;</span><span class="add-favorite pull-right glyphicon glyphicon-plus"></span></p>
```

Comment:
In need of separator lines above and below the "poem-info" class to separate it visually from the rest of the poem data.